dashboard/test/models/user_test.rb

```diff
@@ -2024,6 +2024,65 @@ def mock_geocoder_result(result)
     end
   end
 
+  class AssignedCoursesAndScripts < ActiveSupport::TestCase
+    setup do
+      @student = create :student
+      @course = create :course, name: 'course'
+    end
+
+    test "it returns assigned courses" do
+      teacher = create :teacher
+      section = create :section, user_id: teacher.id, course: @course
+      Follower.create!(section_id: section.id, student_user_id: @student.id, user: teacher)
+
+      assigned_courses = @student.assigned_courses
+      assert_equal 1, assigned_courses.length
+
+      assert_equal 'course', assigned_courses[0][:name]
+    end
+
+    test "it checks visibility of assigned scripts" do
+      # no assigned scripts
+      assert_equal false, @student.any_visible_assigned_scripts?
+
+      # assigned hidden script
+      hidden_script = create :script, name: 'hidden-script', hidden: true
+      @student.assign_script(hidden_script)
+      assert_equal false, @student.any_visible_assigned_scripts?
+
+      # assigned a hidden script and a visible script
+      visible_script = create :script, name: 'visible-script'
+      @student.assign_script(visible_script)
+      assert_equal true, @student.any_visible_assigned_scripts?
```

Comment:
Unless you specifically want to make sure that it's exactly `true`, you can just use `assert` here (and `refute` above)